### PR TITLE
Make `Server.RunCommand` return response

### PR DESCRIPTION
**Pull request by ITeMbI4**

### CODE REVIEW

File: Exiled.API/Features/Server.cs [hunk 1]
```diff
@@ -248,7 +248,7 @@ public static bool ShutdownRedirect(ushort redirectPort)
         /// </summary>
         /// <param name="command">The command to be run.</param>
         /// <param name="sender">The <see cref="CommandSender"/> running the command.</param>
-        public static void RunCommand(string command, CommandSender sender = null) => GameCore.Console.singleton.TypeCommand(command, sender);
+        public static string RunCommand(string command, CommandSender sender = null) => GameCore.Console.singleton.TypeCommand(command, sender);
```
Comment (louis1706) [inline Exiled.API/Features/Server.cs:251]: Breaking change make obsolete the previous one also can we get if the command has successfully be made or not ?

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Server.cs (hunk 1 above)
Comment (Misaka-ZeroTwo) [reply Exiled.API/Features/Server.cs:251]: > Breaking change make obsolete the previous one also can we get if the command has successfully be made or not ?

Compile Error CS0111 will be thrown if a class contains two member declarations with the same name and parameter types.

Comment (VALERA771) [reply Exiled.API/Features/Server.cs:251]: > > Breaking change make obsolete the previous one also can we get if the command has successfully be made or not ?
> 
> 
> 
> Compile Error CS0111 will be thrown if a class contains two member declarations with the same name and parameter types.

Can be renamed to ExecuteCommand or something else like this. It really will be a breaking change so

Comment (ITeMbI4) [reply Exiled.API/Features/Server.cs:260]: @Misaka-ZeroTwo @louis1706 check please

**Review by louis1706 — APPROVED** [review]

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**